components/LocationComparisonSection/LocationComparisonFilters.tsx

```diff
@@ -74,7 +77,9 @@ const LocationComparisonFilters: FunctionComponent<SpotlightFilterProps> = props
         />
       </FormField>
       <FormField className={props.topicClassName}>
-        <Button className={'button'}>{'Update'}</Button>
+        <Button className={'button'} onButtonClick={onUpdateBtnbClick}>
+          {'Update'}
```

Comment:
Remove the curly braces from around strings.

Reply:
Removed the braces